conf.yaml

```
@@ -1430,35 +1430,50 @@ contents:
     -
         title:      Enterprise Search
         sections:
+          -
+            title:      Enterprise Search Guide
+            prefix:     en/enterprise-search
+            index:      enterprise-search-docs/index.asciidoc
+            private:    1
+            current:    7.7
```

Once #1791 merges, we can update this to `*stackcurrent`. That'll keep the current branch synced with the rest of the stack books.
```suggestion
            current:    *stackcurrent
```

---

After talking to @goodroot, let's wait on doing this until after #1791 is merged. That'll keep dependency to a minimum and keep the CI happy for both PRs.

I'll plan to add this variable after this PR and #1791 are merged.